### ISSUE TITLE
Rules Typos (and filling out the general questions in the template)

### ISSUE THREAD

**Issue by sum1els-com** [jacobsalmela/ascii-planets#24]
##### Version of rulebook and Planet Sheet

ascii_planets_v1.1.0

##### Is the game fun to play? What could be done to make it feel more enjoyable?

It seemed fun, once I understood most of the rules. =]  The feedback below has grown pretty long, I hope it's helpful!

##### What player counts have you played with? How did the game work with those?

Just solo so far

##### How is the interaction / dynamic like in multiplayer games? How could it be improved?

N/A

##### Do some actions or strategies feel overly powerful? Is there something you "always need to do" in order to score high?

Advancing the tech tree as soon as possible to get more dice seems important.  Trade and Culture seem important.

##### Are there actions or strategies that seem weak? Do you find you avoid doing something because it doesn't seem to pay off?

Fighting pirates seems like a much longer ramp with less payoff, so I didn't fight them at all.

##### What is the highest score you have seen in a game? How was this accomplished?

Let me play some more games and get back to you on this.

##### Does the Planet Sheet print correctly (in b&w/color)? Are there any issues when printing it?

I haven't printed it yet; I'm just using Edge as a PDF reader and drawing right on it.

##### Is the the Planet Sheet easy to read and understand? Do you have any suggestions for improvement?

(Note "the the" above.)  Military note about "checked" boxes is confusing, am I supposed to put an M in it or check it?  I suppose it doesn't really matter, since the only place the letters are needed for clarification is on the planet map.  The instructions for population are much clearer, so I'd make it like those.  In fact, I don't see the M symbol mentioned in the rules anywhere.

I'm not sure how to use the check boxes on top of the fortifications, and there doesn't seem to be a [!] where I can indicate fortification building mastery (the rules seem to indicate that the game end corresponds to a [!].)  This could be clearer somehow to indicate both that I have completed at least one fortification in the level and can therefore build in the next level, and also indicate that I have built 6 different fortifications triggering end game and gaining a [!].

I'd like the map key to be closer to the map, but I see it's there for fit reasons.

Entirely missing is any mention of spending currency to advance population, military, culture, technology, or happiness; I entirely forgot about that when I played!

##### Is the rulebook easy to understand? Are any of the rules unclear?

Frankly I think using the term "Fortification" is a bit confusing, since that typically indicates some kind of defensive structure.  I suppose the academy might be armored, but it's still awkward.  "Asset" might be better, or even "structure", but I'd find a writer friend to help come up with a better term.

If I build an Academy and a Factory on a turn when I have no available Astronauts, how do I mark the squares I drew on the map to keep them straight for a later turn when I get an Astronaut to staff it?  Also, the wording about having to staff the fortification as soon as possible is a bit confusing and I had to read it several times to understand what was intended.

Can you establish more than one battleship in a single player's territory?  If so, does a single 6 activate all of your battleships allowing you to activate your opponent's fortifications more than once and giving them two unhappiness?

Can you replace a destroyed battleship?

Do you have to staff a battleship with an Astronaut?

Astronauts can "Be trained as Pilots when activating Starships or Battleships" - if you mean when activating a Starship or activating a Battleship to activate Starships, maybe just say "when activating Starships."  Is there some other time when Battleships consume Astronauts?

A Great Person can be used to "Add 4 to the available dice available to spend for the purpose of Construction."  Does this count as an extra die of value 4, or does it increase the value of a die by 4?  i.e. Can I just use two Great People to construct a Factory, or must I combine it with some other die?  Perhaps word this "Act as a die of value 4 for the purpose of Construction" or "Add 4 to the value of one die when used for Construction."

##### Are there any typos in rulebook / Planet Sheet?

Rules p.9 "or constructed each Level III Fortificationtype" needs a space.  Oh, and I don't need to build every fortification type, I just need to "construct a Battleship and a Space Station"?  Maybe just write that instead?

Rules p.6 "Establish a Battleship on another player’s territory where you have Reach to." rephrase to "Establish a Battleship on another
player’s territory to which you have Reach."

Rules p. 5 "Seating order: players sitting next to one another always have Reach to each other after you build at staff least one Starship" This sounds like if I build a starship, everyone will have Reach to their neighbors.  The word "always" is confusing, since I have to do something to enable this, so it isn't always.  The word "at" should be "and".  Perhaps rephrase this to " Seating order: you will have Reach to the players sitting next to you after you build and staff at least one Starship."

**Comment by jacobsalmela** [comment]
Thank you so much for the detailed notes!  I will address these.

**Comment by sum1els-com** [comment]
Continuing my job as editor...  ;)

"Activating can be done whenever you have enough die to spend and the Fortification is already Staffed. Activating puts the Fortification to use to generate it’s ability (see ??, p.??)"

Several issues in this part:
- "enough die" is not correct, since the word "enough" should have a plural like "dice" after it.  In this case though, I think changing "enough" to "a" would be better.
- "it's" means "it is" where "its" means "belongs to it".  I remember which to use by thinking "his, hers, its" vs. "he's, she's, it's".
-  Either remove "(see ??, p.??)" or choose an actual reference.

**Comment by sum1els-com** [comment]
Oh, and I'm not sure how granular you want the issues to be in git hub so you can track issues.  I figure much of this is correcting typos and improving language which you'll probably want to do all at once, so I've just lumped it into the same issue.  If you want me to make new issues for things just let me know.  Once you publish a new revision I'll make new issues either way.
